upload/system/library/emshelper.php

```diff
@@ -427,7 +424,7 @@ public function loadCallbackFunction($paymentMethod)
             if ($emsOrder['status'] == 'completed') {
                 $paymentMethod->response->redirect($this->getSucceedUrl($paymentMethod, $orderInfo['order_id']));
             } elseif ($emsOrder['status'] == 'processing' || $emsOrder['status'] == 'new') {
-                $paymentMethod->response->redirect($paymentMethod->emsHelper->getProcessingUrl($paymentMethod));
+                $paymentMethod->response->redirect($this->getProcessingUrl($paymentMethod));
```

Comment:
There is also such a call to internal functions in the helper:
$paymentMethod->emsHelper->getOrderStatus